src/gui/nav/ViewSlider.ts

```diff
@@ -48,17 +46,32 @@ export class ViewSlider implements Component<ViewSliderAttrs> {
 	private _busy: Promise<unknown>
 	private _parentName: string
 	private _isModalBackgroundVisible: boolean
+	private resizeListener: windowSizeListener = () => this._updateVisibleBackgroundColumns()
+	private handleHistoryEvent = () => {
```

Comment:
would be great to unify it with `handleBackPress()` but I guess it isn't possible at the moment at least